### DISCUSSION TITLE
Allow disabling "Mark as answer" for specific categories

### DISCUSSION THREAD

**Discussion by heyhippari** [community/community#22] [Discussions] [answered]
# Why ?

Not all types of discussions are question-answers.

The current system seems very geared to Q&A, but "answers" don't apply to brainstorming discussions or design discussions that teams could wish to have through Github (in order for them to be archived next to the project's code).

It would be useful to be able to disable the "answers" feature on a per-category basis, in order to make the discussion process neater for these use-cases.

# How ?

We could imagine an additional checkbox when editing a category, which would disable the feature when checked.

**Comment by becca** [comment, marked as answer]
👋 this is currently possible. When editing a category you can disable or enable the `mark as answer` functionality

<img width="762" alt="Screen Shot 2020-10-19 at 4 46 04 PM" src="https://user-images.githubusercontent.com/23545426/96515225-96088800-122a-11eb-9538-d3bce271f784.png">

**Reply by heyhippari** [reply]
Oh :smile: I didn't pay attention enough, it seems.

Great feature so far, congrats to the team.

**Reply by alexfinnarn** [reply]
Awesome! I was going to request this feature as well. I missed the edit button for categories kind of like I'm always hunting down the "edit/create labels" button in GH UI. 

I would add that the default "Ideas" category should have answers disabled, IMHO. I want to use that for discussion on the main topic and upvote of the idea on the list page. Letting people mark an idea as "answered" weirds out that plan.
